scout/commands/update/panel.py

```diff
@@ -46,10 +60,47 @@ def panel(panel, version, update_date, update_version):
             LOG.warning(err)
             click.Abort()
 
+    # Any mintainer updates?
+
+    new_maintainer = None
+    if add_maintainer:
+        user_obj = adapter.user(user_id=add_maintainer)
+        if not user_obj:
+            # Check if maintainers exist in the user database
+            LOG.warning(
+                "Maintainer user id {0} does not exist in user database".format(
+                    add_maintainer
+                )
+            )
+            click.Abort()
```

Comment:
```suggestion
            raise click.Abort
```
Otherwise the function will continue, a test for that perhaps :)

Reply:
Hehe, I mistook click.Abort for click.abort() - not a brilliant naming scheme..

Reply:
Yes I have done the same thing. I mixed it up with `context.abort()`